<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistroForm</class>
 <widget class="QWidget" name="RegistroForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Nombres:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lneName">
         <property name="placeholderText">
          <string>Ingrese dos Nombres...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Apellidos: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lneLast">
         <property name="placeholderText">
          <string>Ingrese dos apellidos...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Nota 1:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbxNot1">
         <property name="maximum">
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Nota 2:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="sbxNot2">
         <property name="maximum">
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
